src/screens/game-setup/GameSetup.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react';
+import { Button, SafeAreaView, Switch, Text, TextInput, View } from "react-native";
+
+const GameSetup = () => {
+  const [playerAName, setPlayerAName] = useState('');
+  const [playerBName, setPlayerBName] = useState('');
+  const [scoringMethod, setScoringMethod] = useState(false);
```

Comment:
Can only ever be American scoring or English scoring. Would like to make this more readable through the use of an enum.